src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -180,6 +189,10 @@ export const getResultMetadata = async function (context: ActionContext<ModelCal
 
         selectFilterDefaults(data, commit, PlottingSelectionsMutations.updateBarchartSelections)
         commit(ModelCalibrateMutation.Calibrated);
+
+        const indicators = data.plottingMetadata.barchart.indicators;
+        await fetchFirstNIndicators(dispatch, indicators, 5);
+
```

Comment:
Shouldn't this be happening before the commit of `ModelCalibrateMutation.Calibrated`? I think that's setting the flag that the stepper uses to permit the user advancing to the model outputs page. We maybe get away with it because of awaits, but conceptually it feels like this should come first. 

Oh, in fact I did just try it with a throttled network, and if I clicked on Model outputs step as soon as it was available, I did briefly see no data on the barchart.

Reply:
youre right! my bad, should be fixed now